dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -343,16 +352,19 @@ public String getContent(String mainDescName, String mainDescriptor, Map<String,
             Files.asCharSink(tempMainDescriptor, StandardCharsets.UTF_8).write(mainDescriptor);
 
             WdlBridge wdlBridge = new WdlBridge();
-            wdlBridge.setSecondaryFiles((HashMap<String, String>)secondaryDescContent);
+            final Map<String, String> pathToContentMap = secondarySourceFiles.stream()
+                    .collect(Collectors.toMap(SourceFile::getAbsolutePath, SourceFile::getContent));
+            wdlBridge.setSecondaryFiles(new HashMap<>(pathToContentMap));
 
             // Iterate over each call, grab docker containers
-            Map<String, String> callsToDockerMap = wdlBridge.getCallsToDockerMap(tempMainDescriptor.getAbsolutePath());
+            // TODO -- mainDescName is probably not right!!!
```

Comment:
elaborate (i.e. what is it being set to and what should it be)

Reply:
Sorry, this was a leftover comment when first looking at it. The value is correct; removing comment.